application/app/views/layouts/nav/_reset_button.html.erb

```diff
@@ -0,0 +1,28 @@
+<!-- Warning note above reset -->
+<li class="px-3 pb-3">
+  <small class="text-warning d-block fw-semibold">
+    <i class="bi bi-exclamation-triangle-fill me-1"></i>
```

Comment:
I miss the accessibility tags like `aria-hidden=true` and the alternative text for accessibility devices